python/tvm/script/builder/tir/prim_func_frame.py

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""TVM Script TIR Prim Func Frame"""
+from tvm._ffi import register_object as _register_object
+
+from tvm.tir.expr import Var
+from tvm.tir.buffer import Buffer
+
+
+from . import _ffi_api
+from .base import TIRFrame
+
+from typing import Union
+
+
+@_register_object("script.builder.tir.PrimFuncFrame")
+class PrimFunc(TIRFrame):
+    def __init__(self, name) -> None:
+        self.__init_handle_by_constructor__(_ffi_api.PrimFuncFrame, name)
```

Comment:
let's remove this constructor and introduce `def prim_func(name: str) -> PrimFunc` explicitly to be consistent with C++ side


```suggestion
class PrimFuncFrame(TIRFrame):
    def __init__(self, name) -> None:
        self.__init_handle_by_constructor__(_ffi_api.PrimFuncFrame, name)
```